notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -347,6 +357,18 @@ export default {
           breadcrumb: this.note.breadcrumb,
           toBePublished: this.note.toBePublished,
         };
+        const childContainer = '<div id="note-children-container" class="navigation-img-wrapper" contenteditable="false"><figure class="image-navigation" contenteditable="false">'
```

Comment:
This constant definition could be done under the check of line 366 since it is not used anywhere else